ush/detect_machine.sh

```diff
@@ -38,6 +38,13 @@ case $(hostname -f) in
   *) MACHINE_ID=UNKNOWN ;;  # Unknown platform
 esac
 
+if [[ ${MACHINE_ID} == "UNKNOWN" ]]; then 
+   case ${PW_CSP:-} in
+      "aws" | "google" | "azure") MACHINE_ID=noaacloud ;;
+      *) MACHINE_ID=UNKNOWN ;; # Unknown platform
```

Comment:
```suggestion
      *) PW_CSP="UNKNOWN"
```
`MACHINE_ID` already is `UNKNOWN`, so perhaps a better option would be to put `PW_CSP` as "UNKNOWN" since that is the check in this case block?